src/garage/tf/algos/__init__.py

```diff
@@ -5,14 +5,17 @@
 from garage.tf.algos.dqn import DQN
 from garage.tf.algos.erwr import ERWR
 from garage.tf.algos.npo import NPO
+from garage.tf.algos.npo2 import NPO2
 from garage.tf.algos.ppo import PPO
+from garage.tf.algos.ppo2 import PPO2
```

Comment:
i would prefer to just delete the old one, as long as tests still pass and benchmarks look good

Reply:
Sure

Reply:
the same goes with all of the primitives -- having DistributionFooPolicy/DistributionFooPolicy2 will just create confusion.

If you don't have the time to run the benchmarks before merging this, then feel free to merge it and send a PR to delete/rename everything later.